drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/cloned/feel/BuiltInType.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.scenariosimulation.kogito.client.cloned.feel;
+
+/*
+TODO {gcardosi} restore/replace them
+import java.time.Duration;
```

Comment:
@gitgabrio is it a real TODO? Can I remove the comments in this class?

Reply:
@yesamer 
I'm not sure. Server-side implementation use a lot of different "date/time" formats I could not easily reproduce. @tarilabs Wdyt?

Reply:
the easiest imho is to rely on FEEL builtin for anything date/time related, that is using the builtin functions for temporal, or the DMNv1.3 AT-LITERAL constructs, instead of having to supply the correct instantiation externally?

Reply:
@tarilabs @yesamer 
Whatever it work is fine, but it must be available "client-side"  - GWT-friendly only